src/ivoc/apwindow.cpp

```diff
@@ -257,15 +241,6 @@ void DismissableWindow::name(const char* s) {
         {
             SetWindowText(hw, s);
         }
-    } else
```

Comment:
I think the original was a bug.  It needs examination in the context of a windows build.

Reply:
Since that logic goes back to the beginning of the git log. And it is relevant to Windows and not MAC,  I guess it is best
to keep it for this PR. I'll look into it when I have a windows development environment available.